### PR TITLE
Feat/provisional evidence 20240429

### PR DESCRIPTION
**Pull request by e-t-k**
Adds Provisional Evidence fields + GUI + small gui changes.

Adds four fields to the variant table in the backend:
    Provisional_Evidence_Code_Popfreq
    Provisional_Evidence_Description_Popfreq
    Provisional_Evidence_Code_Bioinfo
    Provisional_Evidence_Description_Bioinfo

For the Popfreq fields, adds a sub-tile on to the Allele Frequency Reference Sets Tile which displays their information. (Bioinfo fields currently unused.)

Also adds a few miscellaneous GUI changes: (see 9128335 ):
- Removes links to new/changed variants on release pages
- Remove three logos from frontpage
- Removes Clinical Significance (BIC) tile

### CODE REVIEW

File: website/content/help/research/allele-frequency-gnomad.md
Comment (melissacline) [inline website/content/help/research/allele-frequency-gnomad.md:1]: Is this a typo correction?  Good catch!

**Review by melissacline — APPROVED** [review]
So far, all of these changes make sense!  There's a lot of work here, and it's great to see these coming together!

**Comment by e-t-k** [comment]
@melissacline Added the text per #1507 - ready for final review and/or merge!

**Review by melissacline — APPROVED** [review]
Looks good!

**Pull request merged**